resources/ext-zib-MedicationContraIndication-ReasonClosure.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="ext-zib-MedicationContraIndication-ReasonClosure" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/ext-zib-MedicationContraIndication-ReasonClosure" />
+  <name value="ExtZibMedicationContraIndicationReasonClosure" />
+  <title value="Ext Zib MedicationContraIndication ReasonClosure" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="An extension to provide a textual explanation of why the medication contraindication has been closed." />
+  <purpose value="This extension represents the ReasonClosure of [MedicationContraIndication](https://zibs.nl/wiki/MedicationContraIndication-v1.0(2020EN))" />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <kind value="complex-type" />
+  <abstract value="false" />
+  <context>
+    <type value="element" />
+    <expression value="Flag" />
+  </context>
+  <type value="Extension" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Extension" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Extension.url">
+      <path value="Extension.url" />
+      <fixedUri value="http://nictiz.nl/fhir/StructureDefinition/ext-zib-MedicationContraIndication-ReasonClosure" />
+    </element>
+    <element id="Extension.value[x]">

```

Comment:
Hier nog de mapping toevoegen? Volgens mij hadden we afgesproken dat voor specifieke exensies in de extensie kan.